content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -56,13 +56,13 @@ A cluster administrator creates a number of PVs. They carry the details of the r
 #### Dynamic
 When none of the static PVs the administrator created matches a user's `PersistentVolumeClaim`,
 the cluster may try to dynamically provision a volume specially for the PVC.
-This provisioning is based on `StorageClasses`: the PVC must request a
+This provisioning is based on `StorageClass` objects: the PVC must request a
```

Comment:
How about removing "This...objects:" and rephraing the next part:

The <link>storage class</link> must exist before you try to dynamically provision storage."